pkg/cli/config/config.go

```diff
@@ -112,7 +140,11 @@ func (config BackplaneConfiguration) testHTTPRequestToBackplaneAPI() (bool, erro
 		http.DefaultTransport = &http.Transport{Proxy: http.ProxyURL(proxyURL)}
 	}
 
-	req, err := http.NewRequest("HEAD", config.URL, nil)
+	url, err := config.GetBackplaneURL()
+	if err != nil {
+		return false, err
+	}
+	req, err := http.NewRequest("HEAD", url, nil)
```

Comment:
Let's use `http.Head()` instead, a refinement to our codebase.

```suggestion
	req, err := res, err := http.Head(url)
```

Reply:
Also make it concise like the prev comment

Reply:
This change may cause to lint failure

```
pkg/cli/config/config.go:141:16: G107: Potential HTTP request made with variable url (gosec)
                if _, err := http.Head(url); err != nil {
```
https://stackoverflow.com/questions/70281883/golang-untaint-url-variable-to-fix-gosec-warning-g107

Reply:
How about we skip the lint check by adding `//nolint`?

Reply:
I think we can use `//nolint`  but we didn't use it before in bp-cli .. finally I didn't touch that function and kept it as it was .. 
but if you have a strong opinion we will change it with `//nolint`